modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -603,4 +603,9 @@ private static class ParentIdAwareDescriptor<T> {
             this.id = id;
         }
     }
+
+    /** List of reserved schema names. */
+    public static List<String> reservedSchemas() {
+        return List.of(SYSTEM_SCHEMA_NAME);
+    }
```

Comment:
```suggestion
    public static boolean isSystemSchema(String schemaName) {
        return List.of(SYSTEM_SCHEMA_NAME).contains(schemaName));
    }
```
Also, a list can be cached.
I'd moved the method to CatalogUtils or made it non-static.

Reply:
done